CMakeLists.txt

```diff
@@ -1126,15 +1126,6 @@ if(SWIFT_INCLUDE_TOOLS)
   # Refer to the large comment above the add_subdirectory(stdlib) call.
   # https://bugs.swift.org/browse/SR-5975
   add_subdirectory(tools)
-
-  # Localization targets are configured in a way that assume the swift
-  # frontend is being built, so trying to include them for other builds
-  # (like stdlib) fail!
-  #
-  # Diagnostics information is only useful for the frontend compiler
-  # anyway, so let's only include it if the compiler is being built,
-  # which at the moment seems like if SWIFT_INCLUDE_TOOLS is defined.
-  add_subdirectory(localization)
```

Comment:
I think it makes more sense to keep `localization` directory - it would make cmake drive converter and serialization of the en.db. Instead you can remove `localization/diagnostics` directory for now.

Reply:
So we will have a completely empty `/localization` directory here in the repository?

Reply:
Or do you mean it will only have the CMake file that serializes `en.yaml`

Reply:
It's not going to be empty, it's going to have CMakeLists.txt and you can keep diagnostics directory there.

Reply:
I think it make sense to remove this `COMMAND` 
```cmake
  COMMAND
    "${SWIFT_NATIVE_SWIFT_TOOLS_PATH}/swift-serialize-diagnostics"
      --input-file-path ${CMAKE_BINARY_DIR}/share/swift/diagnostics/en.yaml
      --output-directory ${CMAKE_BINARY_DIR}/share/swift/diagnostics/
```
from https://github.com/apple/swift/blob/master/localization/CMakeLists.txt.
WDYT?

Reply:
It has to stay, build and toolchain should have both yaml and db files available.